modules/ROOT/pages/_partials/task-view-governed-apis-for-rulesets.adoc

```diff
@@ -0,0 +1,27 @@
+// Used in docs-exchange/asset-details.adoc and docs-api-governance-documentation/view-conformance-status-in-exchange.adoc
+
+In a ruleset's *Governed APIs* page, you can view the list of centrally-governed APIs to which the ruleset applies. 
+
+Developers can use the *Governed APIs* list to help them improve the quality of the rulesets and APIs they develop as follows: 
+
+* When their APIs fail a ruleset, they can see which other APIs are successfully conforming to the ruleset so that they can fix their APIs.
+* When they are discovering available rulesets to use for their API, they can see which APIs are conformant to a specific ruleset to evaluate how helpful it is.
```

Comment:
```suggestion
* When they are discovering available rulesets to use for their API, they can see which APIs conform to a specific ruleset to evaluate how helpful it is.
```

Reply:
I suggest changing "are conformant" to "conform to" because it is simpler and more direct. Your choice. I won't mark it again.

Reply:
How helpful what is - the API or ruleset?

Reply:
This edit would change the meaning. They are in the process of discovering available rulesets in Exchange, and we use the words conformant or noncorformant very specifically for API Governance. Let me think about this one.